inc/custom-css.php

```diff
@@ -28,7 +28,23 @@ function twentytwenty_generate_css( $selector, $style, $value, $prefix = '', $su
 			return;
 		}
 
-		$return = sprintf( '%s { %s: %s; }', $selector, $style, $prefix . $value . $suffix );
+		if ( ! is_array( $selector ) ) {
+			$selector = array_map( 'trim', explode( ',', $selector ) );
+		}
+
+		$selector = apply_filters( 'twentytwenty_generate_css_selector', $selector );
```

Comment:
Can we come up with a good way to add context to this filter? For example, we could move the filters in `twentytwenty_get_customizer_css`, such as `apply_filters( 'twentytwenty_buttons_targets_front_end', ...`, to make the generated CSS universally filterable from the `generate_css_selector` filter. We need a way to pass the `buttons_targets_front_end` context to this new filter, then we can remove the redundant filter in the theme.

Reply:
We could probably add a new function argument `$context` and use it in the filter name or a second param...

`apply_filters( 'twentytwenty_generate_css_selector', $selector, $context )`
or
` apply_filters( 'twentytwenty_generate_css_selector_' . $context, $selector )`

Reply:
Or just remove the function completely and use CSS Custom Properties for those things that need generating.